DebugSwift/Sources/Settings/DebugSwift.swift

```diff
@@ -5,21 +5,16 @@
 //  Created by Matheus Gois on 16/12/23.
 //
 
-import CoreLocation
 import UIKit
 
 public enum DebugSwift {
-    public static func setup() {
-        UIView.swizzleMethods()
-        UIWindow.db_swizzleMethods()
-        URLSessionConfiguration.swizzleMethods()
-        CLLocationManager.swizzleMethods()
-        LogIntercepter.shared.start()
+    public static func hideFeatureByIndexAndSetup(indexArr: [Int]?) {
```

Comment:
we can improve this function name to:

`public static func setup(hideFeatureByIndex indexArr: [Int]?)`

In this way, we can explicit the conditional setup for only to some features.

Reply:
Using Enum, it will be:

```swift
/// Set up the debugging features.
/// - Parameter features: A variable that defines what should be enabled.
public static func setup(features: DebugSwiftFeatures...) {
    // Implementation goes here
}
```

In use:

```swift
DebugSwift.setup(features: .networking, .app)
```

Reply:
Thank you for your advice, I will make the improvements as soon as possible 🚀.